medicaltorch/transforms.py

```diff
@@ -140,13 +155,13 @@ def __init__(self, size, labeled=True):
 
     @staticmethod
     def propagate_params(sample, params):
-        input_metadata = sample['input_metadata']
+        input_metadata = sample['input_metadata'][0]
         input_metadata["__centercrop"] = params
         return input_metadata
 
     @staticmethod
     def get_params(sample):
-        input_metadata = sample['input_metadata']
+        input_metadata = sample['input_metadata'][0]
```

Comment:
Index[0] --> first channel? Would it be a problem if the params (ie cropping coords) are not the same across channel?

Actually, not sure this case could happen. At least, will not happen with both `CenterCrop2D` and `ROICrop2D` since we currently have only one GT for all channels.

Reply:
I've made the assumption that all input_metadata are the same. Maybe I should change that?

Reply:
I am just thinking: if the 2 contrasts do not have the same initial size (case scenario of Andréanne), then the cropping info will not be the same across channels.
Slice index, filename...etc. would also be different I think --> to double check.
If that 's the case, we should be carefull with that :/

Reply:
Good point effectively... need to check that!

Reply:
Any updates here?

Reply:
I guess you're right, I'll fix it now!